R/build.R

```diff
@@ -45,13 +45,12 @@
 #'
 #' ### Setting development mode
 #'
-#' The development `mode` of a site controls where the site is built,
-#' the colour of the package version in the navbar, the version tooltip,
-#' and whether or not the site is indexed by search engines. There are
-#' four possible modes:
+#' The development `mode` of a site controls where the site is built
+#' and the styling of the site (i.e. the colour of the package version in the 
+#' navbar, the version tooltip), and whether or not the site is indexed by 
```

Comment:
I can't hang a comment off of line 61, but I think it might help to add a sentence to the bullet point about `mode: release`.

"Site is styled like a released package, even if the content is for an unreleased or development version."